test-cypress/integration/sparql/sparql-error-handling.spec.js

```diff
@@ -71,10 +71,10 @@ describe.skip('Error handling', () => {
         SparqlSteps.getShowLessExceptionMessage().should('not.exist');
     });
 
-    const stubErrorResponse = (statusCode, errorMessage) => {
-        cy.intercept('POST', '/repositories/' + repositoryId, {
+    const stubErrorResponse = (repositoryId, statusCode, errorMessage) => {
```

Comment:
We introduced stubbing functions in the yasgui integration branch. Can't we start applying the approach in the main as well?

Reply:
Done